python/vowpalwabbit/sklearn_vw.py

```diff
@@ -344,7 +344,9 @@ def __init__(self,
         if any(x in self.params for x in ext_file_args):
             # fitting will be handled by vw directly
             self.fit_ = True
-            self.passes_ = 1
+            if not self.params.get('passes', None): # To prevent overwriting of passes
+                self.params['passes'] = 1   # default 1 if not given
+            self.passes_ = self.params['passes']
```

Comment:
```suggestion
            self.passes_ = self.params.get('passes', 1)
```

Reply:
Oh, I am so sorry, it went completely out of my mind. Thanks :)